### PR TITLE
cincinnati-operator simplify cmds in scorecard-test

### PR DESCRIPTION
**Pull request by hongkailiu**
/hold

Wait for https://github.com/openshift/cincinnati-operator/pull/202

### CODE REVIEW

**Comment by hongkailiu** [comment]
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/54551/rehearse-54551-pull-ci-openshift-cincinnati-operator-master-scorecard/1816467439370637312

The rehearsal looks good.

/pj-rehearse ack
/hold cancel

File: ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml [hunk 1]
```diff
@@ -97,12 +97,7 @@ tests:
     test:
     - as: scorecard-test
       cli: latest
-      commands: |
-        set -x
-        curl -Lso /tmp/operator-sdk.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/operator-sdk/4.15.21/operator-sdk-v1.31.0-ocp-linux-x86_64.tar.gz
-        ls -al /tmp/operator-sdk.tar.gz
-        tar -xvzf /tmp/operator-sdk.tar.gz --strip=2 -C /tmp
-        /tmp/operator-sdk scorecard bundle -o text --kubeconfig $KUBECONFIG -n default
+      commands: make scorecard-test KUBECONFIG=$KUBECONFIG
```
Comment (wking) [inline ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml:100]: I think you can probably drop `KUBECONFIG=$KUBECONFIG`.  From [GNU Make docs][1]:

> Every environment variable that make sees when it starts up is transformed into a make variable with the same name and value.

And [from POSIX][2]:

> Macro definitions shall be taken from the following sources, in the following logical order, before the makefile(s) are read.
> ...
> 3. The contents of the environment, excluding the MAKEFLAGS and SHELL variables and including the variables with null values.
> ...

[1]: https://www.gnu.org/software/make/manual/html_node/Environment.html
[2]: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html#tag_20_76_13_05

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**